### ISSUE TITLE
Hooks support for Perl

### ISSUE THREAD

**Issue by claviola** [nginx/unit#142]
I see in the changelog that at least "atexit" is supported for Python already:
> *) Feature: Python atexit hook support.

It'd be nice to have that and any other future hooks made available for Perl as well, as uwsgi does.

**Comment by VBart** [comment]
@claviola Could you explain what's not working from Perl application side?

**Comment by claviola** [comment]
The application relies on some hooks uwsgi offers such as `uwsgi::log`, `uwsgi::atexit` and `uwsgi::postfork`. `uwsgi::log` could be replaced by eg. `print STDERR` but not the rest.

Their [docs](https://uwsgi-docs.readthedocs.io/en/latest/Hooks.html) are not too updated, but you can see references to these hooks in the tests, eg: https://github.com/unbit/uwsgi/blob/master/t/perl/test.psgi

**Comment by VBart** [comment]
The cited feature has nothing to do with uwsgi hooks, it's about [standard Python functionality](https://docs.python.org/3/library/atexit.html).

**Comment by claviola** [comment]
Right, I was just describing what I saw in the changelog. Adding a callable like this for perl would also make sense, as handling cleanup is something desirable there too.
